worldedit-core/src/main/java/com/sk89q/worldedit/world/World.java

```diff
@@ -218,6 +220,19 @@ default int getBlockLightLevel(BlockVector3 position) {
      */
     void simulateBlockMine(BlockVector3 position);
 
+    //FAWE start
+
+    /**
+     * Return block drops, as if it were mined.
```

Comment:
Better spelling? Using the superpickaxe on grass e.g. drops grass which isn't a block. Item > block imo.